dev/com.ibm.ws.security.acme/resources/com/ibm/ws/security/acme/resources/AcmeMessages.nlsprops

```diff
@@ -177,3 +177,27 @@ CWPKI2037E.useraction=Provide at least one non-null, non-empty domain in the con
 CWPKI2038W=CWPKI2038W: The ACME service was unable to revoke the certificate with serial number {0}. The error is ''{1}.''
 CWPKI2038W.explanation=The ACME service attempted to revoke the certificate, but was unable to.
 CWPKI2038W.useraction=Review the error message for details on the failure. The certificate will need to be revoked manually.
+
+CWPKI2039E=CWPKI2039E: The DN ''subjectDN'' ({0}) contains a ''cn'' relative distinguished name (RDN) value ({1}) that does not match any of the defined domains.
+CWPKI2039E.explanation= The ''cn'' RDN must match one of the defined domains.
+CWPKI2039E.useraction=Provide a ''subjectDN'' attribute value with a ''cn'' RDN value that matches one of the defined domains or does not have the ''cn'' RDN included.
+
+CWPKI2040E=CWPKI2040E: The ''cn'' relative distinguished name (RDN) was not the first RDN in the ''subjectDN'' configuration attribute.
+CWPKI2040E.explanation=If the ''cn'' RDN is defined in the ''subjectDN'' configuration attribute, then it must be the first RDN.
```

Comment:
**change**
If the ''cn'' RDN is defined in the ''subjectDN'' configuration attribute, then it must be the first RDN.
**to**
If the ''cn'' RDN is defined, it must be the first RDN  in the ''subjectDN'' configuration attribute.

Reply:
Agreed.